assets/scss/components/_button.scss

```diff
@@ -0,0 +1,124 @@
+.button-title {
+  font-size: 22px;
+  font-weight: 400;
+  color: $white;
+  font-family: "";
+  src: url();
+}
+.button-wrapper {
+  height: 61px;
+  width: 190px;
+  position: relative;
+  overflow: hidden;
+  &:hover {
+    .main-button-change {
+      transform: translateY(0);
+    }
+  }
+  .main-button {
+    background: $primaryColor;
+    height: 100%;
+    width: 100%;
+    border: 0;
+    border-radius: 5%;
+  }
+  .main-button-change {
+    position: absolute;
+    left: 0;
+    top: 0;
+    height: 100%;
+    width: 100%;
+    border-radius: 5%;
+    border: 0;
+    color: $primaryColor;
+    background: $darkGray;
+    transform: translateY(100%);
+    transition: 0.25s ease-in all;
+    outline: $white;
+  }
+}
+.button {
+  width: 200px;
+  height: 10px;
+  transition: 0.8s ease all;
+  cursor: pointer;
+  position: relative;
+  transform: translateX(0%);
```

Comment:
Po 1 nie musisz pisać "%"

0 to 0, nie ważne jaką jednostkę podasz

w dodatku każdy element domyślnie nie jest przesunięty względem siebie, więc nic się nie zmieni jeśli usuniesz tą linijkę. 

Chyba że to ma wpływ na animację - to wyprowadź mnie z błędu

Reply:
Masz racje nic nie daje. Zrobione